pagerduty/test_iam_users_have_pd_remote_monitoring_configured.py

```diff
@@ -0,0 +1,22 @@
+
+
+import pytest
+from aws.iam.resources import get_all_users_that_can_access_aws_account
+from pagerduty.helpers import alternate_usernames
+from pagerduty.resources import (
+    users_with_remote_access_escalation_policy_and_extension_configured
+)
+
+
+@pytest.mark.pagerduty
+@pytest.mark.parametrize("iam_user", list(get_all_users_that_can_access_aws_account()))
+def test_bastion_users_have_pd_remote_monitoring_configured(
+    iam_user, users_with_remote_access_escalation_policy_and_extension_configured
```

Comment:
Some `iam_user`s are returning as email addresses, so we'll probably need a bunch of alternates or want to strip trailing `@mozilla.{com,org}`